usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -4243,6 +4304,40 @@ CK_RV ep11tok_generate_key(STDLL_TokData_t * tokdata, SESSION * session,
         goto error;
     }
 
+    if (xts) {
+        RETRY_START(rc, tokdata)
+            rc = dll_m_GenerateKey(&mech2, new_attrs2, new_attrs2_len,
+                                   ep11_pin_blob, ep11_pin_blob_len, blob2,
+                                   &blobsize2, csum, &csum_len,
+                                   target_info->target);
+        RETRY_END(rc, tokdata, session)
+        if (rc != CKR_OK) {
+            rc = ep11_error_to_pkcs11_error(rc, session);
+            TRACE_ERROR("%s m_GenerateKey rc=0x%lx mech='%s' attrs_len=0x%lx\n",
+                        __func__, rc, ep11_get_ckm(tokdata, mech->mechanism),
+                        attrs_len);
+            goto done;
```

Comment:
This should be `goto error;`, not `goto done;`.
Also above in line 4295. This was wrong before already, but lets correct it at both places please.